rust_snuba/src/processors/generic_metrics.rs

```diff
@@ -242,6 +246,8 @@ where
 
     let result: Result<Option<T>, anyhow::Error> = T::parse(msg, config);
 
+    metric!(Timer: "process_messages.messages.size", payload_bytes.len());
```

Comment:
I think the prefix `process_messages.messages` is not necessary, that function doesn't exist here. Please go with something more specific to generic-metrics. `generic_metrics.message.size` seems better

Reply:
done